src/lmio1/ukbd.prom

```diff
@@ -0,0 +1,917 @@
+PROM LOCATION UKBD SUM-CHECK NIL 
```

Comment:
What is this file - a dump of a real PROM made with PROMP (since that seems to be where the header comes from)? Or a PROMP conversion of AS8748's output?

Reply:
I don't know.  I never heard about PROMP but if it writes that header, then maybe its a dump.